### PR TITLE
separated UpdateStudent

### PR DESCRIPTION
**Pull request by betulEZ**

### CODE REVIEW

**Review by MoinMarcell — CHANGES_REQUESTED** [review]
Check your 3 frontend issues first ;-)

File: frontend/src/teacherPage/UpdateStudent.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+export type Props = {
+    open: boolean;
+    handleClose: () => void;
+    handleSave: () => void;
+    name: string;
+    surname: string;
+    changeName: (event: React.ChangeEvent<HTMLInputElement>) => void;
+    changeSurname: (event: React.ChangeEvent<HTMLInputElement>) => void;
+}
+export default function UpdateStudent(props: Props){
```
Comment (MaHambach) [inline frontend/src/teacherPage/UpdateStudent.tsx:29]: `export default function UpdateStudent(props: Readonly<Props>){`

File: frontend/src/teacherPage/UpdateStudent.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                }}
+            >
+                <CloseIcon />
+            </IconButton>
+            <DialogContent dividers>
+                <Typography gutterBottom>
+                    <label>
+                        Name:
+                        <input value={props.name} type="text"  onChange={props.changeName} />
+                    </label>
```
Comment (MaHambach) [inline frontend/src/teacherPage/UpdateStudent.tsx:57]: I am not sure, if encapsulating input in a label-element is wise. Maybe try something like: 
```
<div>
    <label htmlFor={"name"}>Name:</label>
    <input id={"name"} type={"text"} onChange={props.changName}/>
</div>
```

Comment (betulEZ) [reply frontend/src/teacherPage/UpdateStudent.tsx]: This is working. :)

**Review by MaHambach — APPROVED** [review]
LGTM

**Pull request merged**